tests/gis_tests/inspectapp/tests.py

```diff
@@ -62,6 +62,15 @@ def test_3d_columns(self):
             self.assertIn('line = models.GeometryField(', output)
             self.assertIn('poly = models.GeometryField(', output)
 
+    def test_indexes(self):
+        out = StringIO()
+        call_command('inspectdb', table_name_filter=lambda tn: tn.startswith('inspectapp_indexes'), stdout=out)
+        output = out.getvalue()
+        self.assertIn('        indexes = [', output, msg='inspectdb should generate indexes.')
+        index = "models.Index(fields=['point'], name="
+        self.assertIn(index, output)
+        self.assertIn('        ]', output)
```

Comment:
I am not sure how to resolve that.
Django silently creates indexes for Foreign key fields and text ops index for `CharField` in postgres as well, but I am not sure why it silently creates these indexes.
Should I simply ignore spatial indexes? Is defining multi-column indexes possible?